### PR TITLE
Inventory Spring Boot app on OpenLiberty

### PR DESCRIPTION
**Pull request by Hemankita**

### CODE REVIEW

**Comment by yeekangc** [comment]
@Hemankita, are you suggesting to offer an (alternate) option to run on Open Liberty?

I see a new MD and a new build file.  Would like to ensure I understand you correctly.

FYI that Gradle support for Boost is coming (next week).  So, we can go with that and integrate directly?

**Comment by Hemankita** [comment]
@yeekangc - Thank you for the edits.

Yeah correct. Currently, that is how I did it. Specified it as an alternative option for what we have already.

**Comment by yeekangc** [comment]
Thank you, @Hemankita.

I woud think we can run on Open Liberty "out of the box".  Obviously, this will be easier done when we have Boost in Gradle.

@osowski, what are your thoughts on going with Liberty directly?

File: OpenLiberty.MD [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+```
+
+3. To deploy the Inventory container, run the following commands:
+
+Before building the Docker image, make sure there are no other dockerfiles in your repo. If a dockerfile already exists, please remove it.
+
+```bash
+# Build the Docker Image
+$ mvn clean install 
+
```
Comment (osowski) [inline OpenLiberty.MD:137]: Looks like this is missing a closing triple-backtick for the formatting here.

File: OpenLiberty.MD [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+Once done, a docker image named `liberty-inventory-spring:latest` will be built automatically for you.
+
+# Start the Inventory Container
```
Comment (osowski) [inline OpenLiberty.MD:143]: Missing the opening triple-backtick here.

File: README.md [hunk 3]
```diff
@@ -168,6 +169,10 @@ $ curl http://localhost:8080/micro/inventory
 
 That's it, you have successfully deployed and tested the Inventory microservice.
 
+## Deploy Inventory Application on OpenLiberty
+
+The Spring Boot applications can be deployed on WebSphere application server Liberty as well. When the application is deployed on Liberty, the default server that is embedded as part of the application will be disabled and it will use Liberty instead of it. For instructions on how to deploy it on OpenLiberty, follow the instructions [here](OpenLiberty.MD).
```
Comment (osowski) [inline README.md]: "application server" should be capitalized, as it is a proper name - WebSphere Application Server.

**Review by osowski — COMMENTED** [review]
A few minor formatting edits, but otherwise looks good.  I'd rather keep the existing Spring implementation as the default and then make it very easy to run Spring on Liberty as an alternative.  Keeping the "most likely" path as the default is preferred, with respect to what we see with client applications and what they are looking to build.

File: OpenLiberty.MD (hunk 1 above)
Comment (Hemankita) [reply OpenLiberty.MD:137]: Corrected

File: OpenLiberty.MD (hunk 2 above)
Comment (Hemankita) [reply OpenLiberty.MD:143]: Corrected the format

File: README.md (hunk 3 above)
Comment (Hemankita) [reply README.md]: Changed it to `WebSphere Application Server`

**Review by osowski — APPROVED** [review]
Approved with changes updated.

**Pull request merged**